tutorials/R/Geospatial-skills/primer-raster-r/Introduction-to-Raster-Data-In-R/Introduction-to-Raster-Data-In-R.Rmd

```diff
@@ -106,6 +106,10 @@ library(rgdal)
 
 # set working directory to data folder
 #setwd("pathToDirHere")
+setwd("C:/Users/mccahill/Documents/GitHub/NEON-DS-Field-Site-Spatial-Data/SJER/RGB")
+
+#Cannot set working directory, will use paste0 command for these tutorials
+wd <- ("C:/Users/mccahill/Documents/Github/")
 
```

Comment:
You can add a line here that just says `setwd(wd)` which will set your working directory to the appropriate dir to find the input data